src/main/java/org/opentestsystem/delivery/logging/RabbitConfiguration.java

```diff
@@ -84,6 +85,7 @@ public Queue springCloudBusQueue() {
   }
 
   @Bean
+  @Primary
```

Comment:
Why is it necessary that primary annotations be added to a shared library?

Reply:
Verified that primary annotation is not required.